src/threads/unfollow.ts

```diff
@@ -0,0 +1,31 @@
+import type { APIGatewayProxyHandler } from 'aws-lambda';
+import { dynamodb } from '../dynamodb';
+import { extractTokenData, getConnectionId } from '../utils/parsers';
+import { badRequest, notFound, ok, serverError, unauthorized } from '../utils/responses';
+import { ForumTable } from './table';
+
+const forumTable = new ForumTable(dynamodb);
+
+export const handler: APIGatewayProxyHandler = async event => {
+  const connectionId = getConnectionId(event);
+  if (!connectionId) return badRequest('connectionId is required');
+  const tokenData = extractTokenData(event);
+  if (!tokenData) return unauthorized();
+  const { participantId, itemId, userId } = tokenData;
+
+  try {
+    const [ followEvent ] = await forumTable.getFollowers({
+      participantId,
+      itemId,
+      limit: 1,
+      filters: { userId, connectionId },
+    });
+    if (!followEvent) return notFound();
+
+    await forumTable.removeThreadEvent(followEvent);
+
+    return ok();
```

Comment:
That's strange to broadcast the "follow" without broadcasting the "unfollow". I know, the reason is that "unfollow" is not an event for now... Maybe it should be sent anyway... but that means message type does not correspond to event type 😕... do whatever sound best to you for now, but just keep that in mind.

Reply:
Messages can be something else/more than thread events, I'll add another kind of message `UnfollowEvent` and send it in the unfollow handler. And I'll update the test as well.

Reply:
Fixed